docs/UserGuide.md

```diff
@@ -268,28 +268,19 @@ Format: `find [n/NAME] [s/SCHOOL] [y/YEAR]`
 * At least one of the optional fields must be provided.
 * The order of the optional fields do not matter. e.g `n/Hans s/River Valley` will match `s/River Valley n/Hans`
 * Only full words will be matched. e.g `han` will not match `hans`.
```

Comment:
You will still have to update this line because it is specific to name-search

Reply:
Right! Nice catch!